src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -61,75 +72,109 @@ export default function FacilitiesRadioWidget({
   );
 
   return (
-    <div>
+    <div className="vads-u-margin-top--3">
       <div aria-live="assertive" className="sr-only">
         Showing VA facilities sorted {sortedByText}
       </div>
       {showVariant && (
-        <Select
-          label="Sort facilities"
-          name="sort"
-          onValueChange={type => {
-            recordEvent({
-              event: `${GA_PREFIX}-variant-method-${type.value}`,
-            });
-            updateFacilitySortMethod(type.value);
-          }}
-          options={sortOptions}
-          value={{ dirty: false, value: sortMethod }}
-          includeBlankOption={false}
-        />
+        <>
+          <div className="vads-u-margin-bottom--3">
+            <Select
+              label="Sort facilities"
+              name="sort"
+              onValueChange={type => {
+                recordEvent({
+                  event: `${GA_PREFIX}-variant-method-${type.value}`,
+                });
+                updateFacilitySortMethod(type.value);
+              }}
+              options={hasUserAddress ? sortOptions : sortOptions.slice(1)}
+              value={{ dirty: false, value: sortMethod }}
+              includeBlankOption={false}
+            />
+          </div>
+          {!hasUserAddress && (
+            <p>
+              Note: to show facilities near your home, add your residential
+              address{' '}
+              <NewTabAnchor href="/profile">in your VA profile</NewTabAnchor>.
+            </p>
+          )}
+          {requestingLocationFailed && (
+            <div className="vads-u-padding-top--1">
+              <InfoAlert
+                status="warning"
+                headline="Your browser is blocked from finding your current location."
+                className="vads-u-background-color--gold-lightest vads-u-font-size--base"
+                level="3"
+              >
+                <p>Make sure your browser’s location feature is turned on.</p>
+                <a
+                  onClick={() =>
+                    updateFacilitySortMethod(
+                      FACILITY_SORT_METHODS.distanceFromCurrentLocation,
+                    )
+                  }
+                >
+                  Retry searching based on current location
+                </a>
+              </InfoAlert>
+            </div>
+          )}
+        </>
       )}
-      {displayedOptions.map((option, i) => {
-        const { name, address, legacyVAR } = option?.label;
-        const checked = option.value === value;
-        const isCerner = isCernerLocation(option.value, cernerSiteIds);
-        let distance;
+      {!variantRequestingLocationFailed &&
```

Comment:
Diff due to indentation here. Only change is not displaying the facilities list is variant test feature flag is on and browser location request fails.